app/address/osgb/DbAddress.scala

```diff
@@ -42,7 +42,7 @@ trait Document {
 case class DbAddress(
                       id: String,
                       lines: List[String],
-                      town: Option[String],
+                      town: String,
```

Comment:
Is this generic cleanup?  Seems out of place in a PR to remove country.

Reply:
Yeh - general cleanup.